### PR TITLE
Apps 1434 adding models

### PR DESCRIPTION
**Pull request by ShadiMah**

### CODE REVIEW

**Review by datafranch — APPROVED** [review]

File: adapters/google_gemma_2/27b_it/dataloop.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
+{
+  "name": "google-gemma2-9b",
+  "displayName": "Google Gemma2 9b",
+  "version": "0.1.22",
+  "scope": "public",
+  "description": "Google Gemma2-9b from NIM API",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/27b_it/dataloop.json:6]: Not all sections are updated to "Google Gemma2 27b it"

File: adapters/google_gemma_2/9b_it/dataloop.json [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{
+  "name": "google-gemma2-27b",
+  "displayName": "Google Gemma2 27b",
+  "version": "0.1.22",
+  "scope": "public",
+  "description": "Google Gemma2-27b from NIM API",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/9b_it/dataloop.json:6]: OK. So the json for "Google Gemma2-27b" was replaced with "Google Gemma2-9b from NIM API"

File: adapters/google_gemma_2/27b_it/dataloop.json [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  "codebase": {
+    "type": "git",
+    "gitUrl": "https://github.com/dataloop-ai-apps/nim-api-adapter",
+    "gitTag": "0.1.22"
+  },
+  "components": {
+    "computeConfigs": [
+      {
+        "name": "gemma2-9b-nim-deploy",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/27b_it/dataloop.json:27]: here

File: adapters/google_gemma_2/27b_it/dataloop.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+          }
+        }
+      }
+    ],
+    "modules": [
+      {
+        "name": "gemma2-9b-nim-module",
+        "entryPoint": "main_adapter.py",
+        "className": "ModelAdapter",
+        "computeConfig": "gemma2-9b-nim-deploy",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/27b_it/dataloop.json:45]: here

File: adapters/google_gemma_2/27b_it/dataloop.json [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+            "displayIcon": "",
+            "description": "NIM API Google Gemma2-9b predict items"
+          }
+        ]
+      }
+    ],
+    "models": [
+      {
+        "name": "gemma2-9b-nim",
+        "moduleName": "gemma2-9b-nim-module",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/27b_it/dataloop.json:85]: here

File: adapters/google_gemma_2/9b_it/dataloop.json [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+  },
+  "codebase": {
+    "type": "git",
+    "gitUrl": "https://github.com/dataloop-ai-apps/nim-api-adapter",
+    "gitTag": "0.1.22"
+  },
+  "components": {
+    "computeConfigs": [
+      {
+        "name": "gemma2-27b-nim-deploy",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/9b_it/dataloop.json:27]: here

File: adapters/google_gemma_2/9b_it/dataloop.json [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+            "type": "rabbitmq",
+            "minReplicas": 0,
+            "maxReplicas": 2
+          }
+        }
+      }
+    ],
+    "modules": [
+      {
+        "name": "gemma2-27b-nim-module",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/9b_it/dataloop.json:42]: here

File: adapters/google_gemma_2/9b_it/dataloop.json [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+            "displayIcon": "",
+            "description": "NIM API Google Gemma2-27b predict items"
+          }
+        ]
+      }
+    ],
+    "models": [
+      {
+        "name": "gemma2-27b-nim",
+        "moduleName": "gemma2-27b-nim-module",
```
Comment (OfirDataloopAI) [inline adapters/google_gemma_2/9b_it/dataloop.json:85]: here

File: adapters/meta_llama3_70b_instruct/dataloop.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "nim-api-llama3-70b-instruct-meta",
-  "displayName": "LLAMA3 NIM API",
+  "displayName": "LLAMA3 70B Instruct",
```
Comment (OfirDataloopAI) [inline adapters/meta_llama3_70b_instruct/dataloop.json:3]: Notice that the modules are still named with **NIM**

**Pull request merged**